File: C/gatos.c
#include <stdio.h>
#include <string.h>

int main(){
    char nome[25];
    double salario,gatos;    
    printf("Qual seu nome?\n");
    scanf("%s", &nome);
    printf("Qual salario?\n");
    scanf("%lf", &salario);
    gatos = ((salario*0.3)/150);
    printf("Ola %s, voce pode ter %.f gatos!", nome, gatos);
}

File: js/gatos.js
function gatos(salary){
    let cats = (salary*0.3)/150
    return cats.toFixed(0)
}

let input = require('fs').readFileSync('stdin', 'utf8')
let lines = input.split('\n')
let nome = lines[0].trim()
let salario = Number(lines[1])
console.log(`Olá ${nome}, você pode ter ${gatos(salario)} gatos!`)